Backup script: `TypeError: LinkExtractionRenderer.link() got an unexpected keyword argument 'url'`

```
[8](https://github.com/marek22k/net4people-bbs-backup/actions/runs/21288145599/job/61274394775#step:5:7519)
https://api.github.com/repos/net4people/bbs/labels/Belarus => 200 OK 2484/5000
https://api.github.com/repos/net4people/bbs/labels/Brazil => 200 OK 2485/5000
https://api.github.com/repos/net4people/bbs/labels/Cameroon => 200 OK 2486/5000
https://api.github.com/repos/net4people/bbs/labels/Canada => 200 OK 2487/5000
https://api.github.com/repos/net4people/bbs/labels/China => 200 OK 2488/5000
https://api.github.com/repos/net4people/bbs/labels/Cuba => 200 OK 2489/5000
https://api.github.com/repos/net4people/bbs/labels/Egypt => 200 OK 2490/5000
https://api.github.com/repos/net4people/bbs/labels/Ethiopia => 200 OK 2491/5000
https://api.github.com/repos/net4people/bbs/labels/Germany => 200 OK 2492/5000
https://api.github.com/repos/net4people/bbs/labels/India => 200 OK 2493/5000
https://api.github.com/repos/net4people/bbs/labels/Indonesia => 200 OK 2494/5000
https://api.github.com/repos/net4people/bbs/labels/Iran => 200 OK 2495/5000
https://api.github.com/repos/net4people/bbs/labels/Japan => 200 OK 2496/5000
https://api.github.com/repos/net4people/bbs/labels/Kazakhstan => 200 OK 2497/5000
https://api.github.com/repos/net4people/bbs/labels/Malaysia => 200 OK 2498/5000
https://api.github.com/repos/net4people/bbs/labels/Myanmar => 200 OK 2499/5000
https://api.github.com/repos/net4people/bbs/labels/Nigeria => 200 OK 2500/5000
https://api.github.com/repos/net4people/bbs/labels/Norway => 200 OK 2501/5000
https://api.github.com/repos/net4people/bbs/labels/Pakistan => 200 OK 2502/5000
https://api.github.com/repos/net4people/bbs/labels/reading%20group => 200 OK 2503/5000
https://api.github.com/repos/net4people/bbs/labels/Russia => 200 OK 2504/5000
https://api.github.com/repos/net4people/bbs/labels/Saudi%20Arabia => 200 OK 2505/5000
https://api.github.com/repos/net4people/bbs/labels/South%20Korea => 200 OK 2506/5000
https://api.github.com/repos/net4people/bbs/labels/Spain => 200 OK 2507/5000
https://api.github.com/repos/net4people/bbs/labels/Sri%20Lanka => 200 OK 2508/5000
https://api.github.com/repos/net4people/bbs/labels/Turkey => 200 OK 2509/5000
https://api.github.com/repos/net4people/bbs/labels/Turkmenistan => 200 OK 2510/5000
https://api.github.com/repos/net4people/bbs/labels/UAE => 200 OK 2511/5000
https://api.github.com/repos/net4people/bbs/labels/Uganda => 200 OK 2512/5000
https://api.github.com/repos/net4people/bbs/labels/USA => 200 OK 2513/5000
https://api.github.com/repos/net4people/bbs/labels/Uzbekistan => 200 OK 2514/5000
https://api.github.com/repos/net4people/bbs/labels/Vietnam => 200 OK 2515/5000
https://api.github.com/repos/net4people/bbs/labels/Zimbabwe => 200 OK 2516/5000
https://avatars.githubusercontent.com/u/33761262?v=4 => 200 OK -/-
https://avatars.githubusercontent.com/u/41267675?v=4 => 200 OK -/-
Traceback (most recent call last):
  File "/home/runner/work/net4people-bbs-backup/net4people-bbs-backup/./backup.py", line 435, in <module>
    backup(owner, repo, db, username, token)
    ~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/net4people-bbs-backup/net4people-bbs-backup/./backup.py", line 341, in backup
    for link in scrape_links():
                ~~~~~~~~~~~~^^
  File "/home/runner/work/net4people-bbs-backup/net4people-bbs-backup/./backup.py", line 338, in scrape_links
    for link in markdown_extract_links(data["body"] or ""):
                ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/net4people-bbs-backup/net4people-bbs-backup/./backup.py", line 161, in markdown_extract_links
    mistune.Markdown(renderer=renderer)(markdown) # Discard HTML output.
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/markdown.py", line 120, in __call__
    return self.parse(s)[0]
           ~~~~~~~~~~^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/markdown.py", line 98, in parse
    result = self.render_state(state)
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/markdown.py", line 56, in render_state
    return self.renderer(data, state)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/core.py", line 257, in __call__
    return self.render_tokens(tokens, state)
           ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/core.py", line 254, in render_tokens
    return "".join(self.iter_tokens(tokens, state))
           ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/core.py", line 251, in iter_tokens
    yield self.render_token(tok, state)
          ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/renderers/html.py", line 38, in render_token
    text = self.render_tokens(token["children"], state)
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/core.py", line 254, in render_tokens
    return "".join(self.iter_tokens(tokens, state))
           ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/core.py", line 251, in iter_tokens
    yield self.render_token(tok, state)
          ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/mistune/renderers/html.py", line 45, in render_token
    return func(text, **attrs)
TypeError: LinkExtractionRenderer.link() got an unexpected keyword argument 'url'
Error: Process completed with exit code 1.
```
Log: https://github.com/marek22k/net4people-bbs-backup/actions/runs/21288145599/job/61274394775#step:5:7559

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,2 +1,2 @@
-mistune >=2.0
-requests >=2.25
+mistune~=2.0
+requests>=2.25

